platform/feature-api/communication/postoffice/README.md

```diff
@@ -22,25 +22,25 @@ The type specification for a simple API can be expressed using standard
 TypeScript techniques:
 
 ```typescript
-import {ObjectEndpointSpec, ValueEndpointSpec} from "@polypoly-eu/postoffice";
+import { ObjectBackendSpec, ValueBackendSpec } from "@polypoly-eu/postoffice";
 
-type SimpleEndpoint = ObjectEndpointSpec<{
-    test1(param1: string): ValueEndpointSpec<number>;
-    test2(param1: string): ValueEndpointSpec<number>;
-    test3(parama: boolean, ...paramb: number[]): ValueEndpointSpec<string>;
-}>
+type SimpleBackend = ObjectBackendSpec<{
+    test1(param1: string): ValueBackendSpec<number>;
+    test2(param1: string): ValueBackendSpec<number>;
+    test3(parama: boolean, ...paramb: number[]): ValueBackendSpec<string>;
+}>;
 ```
 
-By convention, these specifications are called “endpoints”.
+By convention, these specifications are called “backend endpoints”.
 
 Servers can implement this endpoint specification and may choose to use `async`
-methods at any point.  We assume port-authority as a transport layer.
+methods at any point. We assume port-authority as a transport layer.
 
 ```typescript
-import {endpointServer, ServerOf} from "@polypoly-eu/postoffice";
+import {backendServer, ServerOf} from "@polypoly-eu/postoffice";
```

Comment:
OK, this is a naming nit, but then this is a naming PR, so... `backendServer` seems redundant (we already know that the backend is a server). `endpointServer`, somehow, made more sense: it serves endpoints. However, `backend` is a server by itself. You're making `backendEndpointServer` less verbose (and eliminating the repetition of "end" in the process), but even if it's a bit funky, something like `backEndpointServer` might be a bit more clear.

Reply:
I think that in this case, backend just there to give context of where server is, as in it is not external. Which works for me in terms of how understandable it is. Endpoint here might be a little implicit and can cause some confusion when put as backEndpoint or backendEndpoint. However, I don't have a strong preference